AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/AWSDataStorePlugin+DataStoreSubscribeBehavior.swift

```diff
@@ -11,10 +11,25 @@ import Combine
 extension AWSDataStorePlugin: DataStoreSubscribeBehavior {
     @available(iOS 13.0, *)
     public func publisher<M: Model>(for modelType: M.Type)
+        -> AnyPublisher<MutationEvent, DataStoreError> {
+        return publisher(for: modelType.modelName)
+    }
+
+    @available(iOS 13.0, *)
+    public func publisher(for modelName: ModelName)
+        -> AnyPublisher<MutationEvent, DataStoreError> {
+            reinitStorageEngineIfNeeded()
+            // Force-unwrapping: The optional 'dataStorePublisher' is expected
+            // to exist for deployment targets >=iOS13.0
+            return dataStorePublisher!.publisher(for: modelName)
```

Comment:
Let's compose these here rather than cluttering the dataStorePublisher API.
```suggestion
            return publisher()
                .filter { $0.modelName == modelName }
                .eraseToAnyPublisher()
```

Reply:
Fixed